### PR TITLE
Add more preview enhancements

### PR DESCRIPTION
**Pull request by ThomasJClark**
Points and sizes can be previewed, and the background image in blob/line
previews can be disabled

### CODE REVIEW

File: src/main/java/edu/wpi/grip/ui/preview/PointSizeSocketPreviewView.java [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+import org.bytedeco.javacpp.IntPointer;
+
+import static org.bytedeco.javacpp.opencv_core.*;
+
+/**
+ * A {@link SocketPreviewView} for OpenCV points and sizes
+ */
+public class PointSizeSocketPreviewView extends SocketPreviewView<IntPointer> {
+
+    final TextField x, y;
```
Comment (JLLeitschuh) [inline src/main/java/edu/wpi/grip/ui/preview/PointSizeSocketPreviewView.java]: Take care of this please.

File: src/main/java/edu/wpi/grip/ui/preview/BlobsSocketPreviewView.java [hunk 2]
```diff
@@ -27,6 +28,7 @@
     private final ImageView imageView = new ImageView();
     private final Label infoLabel = new Label();
     private final Mat tmp = new Mat();
+    private boolean showInputImage;
```
Comment (JLLeitschuh) [inline src/main/java/edu/wpi/grip/ui/preview/BlobsSocketPreviewView.java]: Default state undefined

Comment (ThomasJClark) [inline src/main/java/edu/wpi/grip/ui/preview/BlobsSocketPreviewView.java]: That's not true, but I'll explicitly initialize it to make it clearer.

File: src/main/java/edu/wpi/grip/ui/preview/LinesSocketPreviewView.java [hunk 3]
```diff
@@ -65,10 +71,16 @@ private void convertImage() {
             // If there were lines found, draw them on the image before displaying it
             if (!linesReport.getLines().isEmpty()) {
                 if (input.channels() == 3) {
```
Comment (JLLeitschuh) [inline src/main/java/edu/wpi/grip/ui/preview/LinesSocketPreviewView.java]: Don't care about this.

File: src/main/java/edu/wpi/grip/ui/preview/BlobsSocketPreviewView.java [hunk 4]
```diff
@@ -60,10 +66,16 @@ private void convertImage() {
             // If there were lines found, draw them on the image before displaying it
             if (!blobsReport.getBlobs().isEmpty()) {
                 if (input.channels() == 3) {
```
Comment (JLLeitschuh) [inline src/main/java/edu/wpi/grip/ui/preview/BlobsSocketPreviewView.java]: Whatever, don't care.

File: src/main/java/edu/wpi/grip/ui/preview/LinesSocketPreviewView.java [hunk 5]
```diff
@@ -31,6 +31,7 @@
     private final ImageView imageView = new ImageView();
     private final Label infoLabel = new Label();
     private final Mat tmp = new Mat();
+    private boolean showInputImage = false;
```
Comment (ThomasJClark) [inline src/main/java/edu/wpi/grip/ui/preview/LinesSocketPreviewView.java]: This isn't true.

@JLLeitschuh Can you disable this bot?  It's kind of annoying.

**Pull request merged**